tachyon/math/polynomials/univariate/radix2_evaluation_domain.h

```diff
@@ -99,12 +99,10 @@ class Radix2EvaluationDomain : public UnivariateEvaluationDomain<F, MaxDegree>,
       NOTREACHED();
     }
     CHECK_EQ(this->size_, static_cast<size_t>(mat.rows()));
-    uint32_t log_n = this->log_size_of_group_;
-    mid_ = log_n / 2;
 
     // The first half looks like a normal DIT.
     ReverseMatrixIndexBits(mat);
-    RunParallelRowChunks(mat, roots_vec_[log_n - 1], packed_roots_vec_[0]);
+    RunParallelRowChunks(mat, roots_vec_.back(), packed_roots_vec_[0]);
```

Comment:
This appears belong to a separate commit (aka appears not relevant to moving `mid`.)

Reply:
Oh this is because computing `mid` depends on computing `log_n`, and `mid` was gone ad `log_n` was gone accordingly.